### PR TITLE
Simplify code

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
### Summary of Changes

Simple change.
Simplify code after https://github.com/joomla/joomla-cms/commit/08a88f00e82bf2f710a26f64d09cc9888d0bea48
### Testing Instructions

Code review.
See https://github.com/joomla/joomla-cms/pull/10886
### Documentation Changes Required

None.

### CODE REVIEW

**Comment by PhilETaylor** [comment]
"Code review" should not be an acceptable testing method... 

Why can this not be covered by unit tests? or are they already?

**Comment by andrepereiradasilva** [comment]
> "Code review" should not be an acceptable testing method... "Code review" as a testing method for PR just ends up in mistakes (ask @brianteeman haha)

As principle i agree. But for this changes. You want to test all URL schemes?

> Why can this not be covered by unit tests? or are they already?

don't know, don't understand unit tests very well.

**Comment by PhilETaylor** [comment]
> There are a lot of our tests that don't really test anything useful 

#facepalm 

Still not a reason not to add "useful" tests :)

**Comment by mbabker** [comment]
If you can figure out what's useful in the unit test suite, or even better decipher `JHtmlTest::testStylesheet()` and `JHtmlTest::testScript()`, you're doing better than me, and I've been trying to fix some of the tests for years.

File: libraries/joomla/form/rule/url.php [hunk 1]
```diff
@@ -94,9 +94,9 @@ public function test(SimpleXMLElement $element, $value, $group = null, Registry
 		}
 
 		// For some schemes here must be two slashes.
-		if (($urlScheme == 'http' || $urlScheme == 'https' || $urlScheme == 'ftp' || $urlScheme == 'ftps' || $urlScheme == 'gopher'
-			|| $urlScheme == 'wais' || $urlScheme == 'prospero' || $urlScheme == 'sftp' || $urlScheme == 'telnet' || $urlScheme == 'git')
-			&& ((substr($value, strlen($urlScheme), 3)) !== '://'))
+		$scheme = array('http', 'https', 'ftp', 'ftps', 'gopher', 'wais', 'prospero', 'sftp', 'telnet', 'git');
```
Comment (dgt41) [inline libraries/joomla/form/rule/url.php]: do you really have to declare a new var $scheme here? I wouldn't mind that array in the in_array() below

Comment (andrepereiradasilva) [inline libraries/joomla/form/rule/url.php]: is not a new var is a reuse of a var a few lines above ;)

**Comment by dgt41** [comment]
I have tested this item :white_check_mark: successfully on 4f8b0728b99ea54d73e6b9bf444d1abd1b90c98f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11817">issues.joomla.org/joomla-cms/11817</a>.</sub>

**Comment by truptikagathara** [comment]
I have tested this item :white_check_mark: successfully on 4f8b0728b99ea54d73e6b9bf444d1abd1b90c98f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11817">issues.joomla.org/joomla-cms/11817</a>.</sub>

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 4f8b0728b99ea54d73e6b9bf444d1abd1b90c98f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11817">issues.joomla.org/joomla-cms/11817</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11817">issues.joomla.org/joomla-cms/11817</a>.</sub>

**Pull request merged**